addons/account/models/chart_template.py

```diff
@@ -1,991 +1,680 @@
 # -*- coding: utf-8 -*-
 
-from odoo.exceptions import AccessError
-from odoo import api, fields, models, Command, _, osv
-from odoo import SUPERUSER_ID
-from odoo.exceptions import UserError, ValidationError
-from odoo.http import request
-from odoo.addons.account.models.account_tax import TYPE_TAX_USE
-from odoo.addons.account.models.account_account import ACCOUNT_CODE_REGEX
-from odoo.tools import html_escape
-
+import ast
+import csv
+from functools import lru_cache
 import logging
+from collections import defaultdict
 import re
 
+from odoo import SUPERUSER_ID, Command, _, api, models
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+from odoo.tools import file_open
+
 _logger = logging.getLogger(__name__)
 
-def migrate_set_tags_and_taxes_updatable(cr, registry, module):
-    ''' This is a utility function used to manually set the flag noupdate to False on tags and account tax templates on localization modules
-    that need migration (for example in case of VAT report improvements)
-    '''
-    env = api.Environment(cr, SUPERUSER_ID, {})
-    xml_record_ids = env['ir.model.data'].search([
-        ('model', 'in', ['account.tax.template', 'account.account.tag']),
-        ('module', 'like', module)
-    ]).ids
-    if xml_record_ids:
-        cr.execute("update ir_model_data set noupdate = 'f' where id in %s", (tuple(xml_record_ids),))
+TEMPLATE_MODELS = (
+    'res.company',
+    'account.account',
+    'account.tax.group',
+    'account.tax',
+    'account.journal',
+    'account.group',
+    'account.reconcile.model',
+    'account.fiscal.position',
+)
+
+TAX_TAG_DELIMITER = '⸾'
```

Comment:
hooooo so that's _the_ utility of this unicode character? I suppose it's safer than a ordinary '|', so keep it wiggly?

Reply:
Since we are using the string of the tags (could be anything), we need something that is very unlikely used